api/src/main/java/org/apache/cloudstack/api/response/ZoneResponse.java

```diff
@@ -344,6 +336,30 @@ public Map<String, String> getResourceDetails() {
         return resourceDetails;
     }
 
+    public Boolean getSecurityGroupsEnabled() {
+        return securityGroupsEnabled;
+    }
+
+    public Boolean getLocalStorageEnabled() {
+        return localStorageEnabled;
+    }
+
+    public Boolean getAllowUserSpecifyVRMtu() {
+        return allowUserSpecifyVRMtu;
+    }
+
+    public Integer getRouterPrivateInterfaceMaxMtu() {
+        return routerPrivateInterfaceMaxMtu;
+    }
+
+    public Integer getRouterPublicInterfaceMaxMtu() {
+        return routerPublicInterfaceMaxMtu;
+    }
+
+    public boolean getNsxEnabled() {
```

Comment:
it is an eternal discussion `is...()` vs. `get..()`, but as this is not a wrapper, isn't `is()` more appropriate?

Reply:
`BeanUtils.copyProperties` expects the names to be like `get...`. If I use `is...()`, it doesn't work.

Reply:
This happened for `localStorageEnabled` & `securityGroupsEnabled` which are `Boolean`. Not sure about `nsxEnabled` which is a primitive type (`boolean`).

Reply:
as discussed off-line; let's use as many primitive as possible and expose those with `is..()` calls. Wrappers are useful only on optional output.